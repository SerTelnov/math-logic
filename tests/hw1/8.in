A1, B1|-A2->B1
A1
B1
A1->A2->B1
A2->B1