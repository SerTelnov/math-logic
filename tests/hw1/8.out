(1) A1 (Предп. 1)
(2) B1 (Предп. 2)
(3) (A1->(A2->B1)) (Не доказано)
(4) (A2->B1) (M.P. 3, 1)